src/components/shared/form_fields/IbanField.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+	<div class="form-field form-field-iban" :class="[ 'form-field-text', { 'is-invalid': showError } ]">
+		<label for="account-number" class="form-field-label">
+			{{ $t( 'donation_form_payment_bankdata_account_iban_label' ) }}
+		</label>
+		<div class="control text-form-input" :class="{ 'has-icons-right': showError }">
+			<input
+				ref="field"
+				name="iban"
+				class="input"
+				id="iban"
+				:class="{ 'is-danger': showError }"
+				type="text"
+				autocomplete="on"
+				:placeholder="$t( 'donation_form_payment_bankdata_account_iban_placeholder' )"
+				:aria-invalid="showError"
+				:aria-describedby="ariaDescribedby"
+				@blur="onBlur"
+				@input="onInput"
+				@paste="onInput"
+			/>
+		</div>
+		<span v-if="showError" class="help is-danger" id="iban-error">{{ $t( 'donation_form_payment_iban_error' ) }}</span>
+		<span class="field-info-message iban-bank-name" v-if="modelValue !== '' && bankName">
+			{{ bankName }} ({{ bic }})
+		</span>
+	</div>
+</template>
+
+<script setup lang="ts">
+
+import { computed, nextTick, onMounted, ref, watch } from 'vue';
+import { useAriaDescribedby } from '@src/components/shared/form_fields/useAriaDescribedby';
+
+interface Props {
+	modelValue: string;
+	showError: boolean;
+	bankName: string;
+	bic: string;
+	ariaDescribedby?: string;
+}
+
+const props = withDefaults( defineProps<Props>(), {
+	ariaDescribedby: '',
+} );
+const emit = defineEmits( [ 'field-changed', 'update:modelValue' ] );
+
+const fieldModel = ref<string>( props.modelValue );
+const field = ref<HTMLInputElement>( null );
+const ariaDescribedby = useAriaDescribedby(
+	computed<string>( () => props.ariaDescribedby ),
```

Comment:
This is the first time I'm noticing this pattern of converting a prop into a Ref. It is a bit confusing and feels weird. I'm wondering if we can increase performance and write more idiomatic Vue by using [toRef](https://vuejs.org/api/reactivity-utilities#toref) instead of `computed`.

Reply:
TS throws warnings when passing a prop straight into a composable, and my default is to use a computed in that case. But yeah I just tested toRef and it works. For some reason we need to not use type parameters with it, and type the return value of the function instead.